modules/salt/pages/formula-saltboot.adoc

```diff
@@ -109,12 +109,66 @@ Leave the other fields empty.
 
 
 
-== Troubleshooting the Saltboot Formula
+== Customize the Client Boot Process
+
+You can change the client boot process using Salt pillars.
+There are two Salt pillars for this purpose, that allow you to change the protocol and server used to the download the image.
+
+* The ``saltboot_download_protocol`` pillar specifies which protocol should be used to download the image to the client.
+This overrides the default protocol specified in the image pillar.
+Allowed values are ``http``, ``https``, ``ftp`` or ``tftp``.
+* The ``saltboot_download_server`` pillar specifies which server to use to download the image.
+This overrides the default hostname specified in the image pillar.
+
+
+
+.Example: Changing the Saltboot Image Download Protocol
+This example changes the protocol used for all clients by editing the ``/srv/pillar/top.sls`` file:
```

Comment:
Same as in example below, it concerns two files. `/srv/pillar/top.sls` and `/srv/pillar/saltboot_proto.sls`.

Reply:
ugh, not sure how i missed that! fixed now :)